examples/deck/example.js

```diff
@@ -565,14 +573,16 @@ export default class Example extends Component<
         ) {
           // Don't log edits that happen as the pointer moves since they're really chatty
           // eslint-disable-next-line
-          console.log('onEdit', editType, featureIndexes, editContext);
+          console.log('onEdit', editType, editContext);
```

Comment:
Unrelated: Use probe.gl log class so that logging can be turned on and off?

Reply:
Agreed.#136